docs/basic-tutorial/basic-features(todolist)/10-local-states/local-states-kr.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: default
+title: 10. 상태(state) 추가하기
+parent: 할 일 목록 앱 만들기
+grand_parent: Basic(한글)
+nav_order: 10
+---
+
+## 상태(State) 추가하기
+
+할 일 목록 추가, 삭제 기능 등을 구현하기 위해서는 상태값을 정의해야 합니다. `todos` 라는 속성으로 상태를 정의해서 할 일 목록의 상태를 관리하겠습니다. `todos` 객체는 아래와 같이 세 가지 속성값을 가집니다.
+
+```
+todos: {id: Number, textValue: string, checked: boolean }
+```
+
+- `id`: 각 목록의 고유 아이디
+- `textValue`: 목록 내용
+- `checked`: 완료 여부. (`true`이면 완료 `false`이면 미완료)
+
+본 튜토리얼에서는 상태 관리를 위해서 훅(hook)을 사용하겠습니다.
```

Comment:
I think linking official link could be help someone who want to find more about `hook`.

Reply:
Yes I will add links on the post.

Reply:
I added official link to the sentence.